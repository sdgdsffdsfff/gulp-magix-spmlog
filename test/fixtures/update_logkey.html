<div data-spm-click="gostr=/*.*.*;locaid=*"></div>