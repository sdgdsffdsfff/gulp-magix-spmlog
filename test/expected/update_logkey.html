<div data-spm-click="gostr=/1.2.3;locaid=*"></div>